### ISSUE TITLE
"Clear submission history" also clears Read Later

### ISSUE THREAD

**Issue by tp0** [ccrama/Slide#2538]
I think this is wrong. It either shouldn't clear, or it should warn the user about it.

(The sitaution is made even more confusing as Read Later isn't included in the settings backup which ["includes history"](http://i.imgur.com/kLXPKIe.png), yet the history clearing feature still counts it as part of the history.)

Slide version: 5.6.5.1-alpha4
Android version: 6.0.1

**Comment by ccrama** [comment]
The settings backup does not include history, and all submissions/read later are stored in an SQLite database on your device. When you clear posts, it empties the database. Might need some better wording as I can see how that is confusing

**Comment by PiwwowPants** [comment]
Would you want the clear action to continue to behave that way? As it is now, it looks like history posts are receiving an empty string prefix.
I'm not completely clear on how the database is handled, but could the prefix not be changed to something like "history" then this be run on clear to only clear history from the db?

`KVStore.getInstance().getByPrefix("history").clear();`

**Comment by ccrama** [comment]
@PiwwowPants that is correct, that would actually be possible and might be a better way of doing this. Will look into it!
